Documentation/ceph-monitoring.md

```diff
@@ -81,7 +81,11 @@ You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
 ## Prometheus Alerts
-To enable prometheus alerts during cluster deployment, make following changes to `cluster.yaml`
```

Comment:
Make an L3 subheader for enabling alerts during cluster deployment and another for enabling alerts after cluster deployment so that readers can see at a glance that they have both of these options documented.